Makefile

```diff
@@ -55,15 +55,7 @@ GRUB_MODULES = \
 	true \
 	video
 
-all:
```

Comment:
everything above can be removed as well given this change.

Reply:
Done.